cmd/root.go

```diff
@@ -245,6 +248,15 @@ func checkForDebugFlag(args []string) bool {
 	return false
 }
 
+func checkForEnableANSIColorFlag(args []string) bool {
```

Comment:
This is not a correct way of reading flags, can't you just parse it and then check its value?

Reply:
The reason for this is because similar to the debug flag, we need to manually parse and get the value of the flag early as in root command the colorCodeConsole is initialised first and then the flags are parsed.

I had tested it without manually parsing it and the flag value wasn't updated until after the console was created.

Reply:
We are currently refactoring the cmd package, then you will be able to parse it properly, without having to read it like that.